### PR TITLE
[ADD] fr_CA translation of the chart of accounts

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by mart-e** [comment]
Hello,

Thanks for the PR. To make your CoA multilang, you need to specify a `spoken_languages` key in your  `account.chart.template` record. See [l10n_be](https://github.com/odoo/odoo/blob/4e0821ae9a6a8389032ef52e3f9f7bdf081cc7ff/addons/l10n_be/account_chart_template.xml#L29) for instance

**Comment by max3903** [comment]
@mart-e Done here: https://github.com/odoo/odoo/pull/9398/files#diff-7742082f5a37d54ad94cab0ed4503b08R17

File: addons/l10n_ca/account_chart_template.xml [hunk 1]
```diff
@@ -11,9 +11,10 @@
     </record>
 
     <record id="ca_en_chart_template_en" model="account.chart.template">
-        <field name="name">Canada - Chart of Accounts for english-speaking provinces</field>
+        <field name="name">Canada - Chart of Accounts</field>
         <field name="currency_id" ref="base.CAD"/>
         <field name="use_anglo_saxon" eval="True"/>
+        <field name="spoken_languages" eval="fr_CA"/>
```
Comment (mart-e) [inline addons/l10n_ca/account_chart_template.xml]: `eval="'fr_CA'"`, it is a list of comma separated string (I know, not obvious)

**Comment by mart-e** [comment]
Thank you, there was still a missing `post_init_hook` command to specify but I added it when merging.
Done at 58f06d6a9c1ad9900da7eafb61715fb18de04527

**Pull request closed without merging**